README.md

```diff
@@ -130,6 +130,8 @@ Next, configure the aspectj-maven-plugin to compile-time weave (CTW) the aws-lam
 <summary><b>Gradle - Java 11+</b></summary>
 
 ```groovy
+        // Make sure you are using gradle >= 8.2.1 
```

Comment:
I'm not sure about the version to use, I used the latest but maybe a 8.x is good enough.